src/xdocs/google_style.xml

```diff
@@ -970,6 +970,13 @@
                   <span class="wrapper inline">
                     <img src="images/ok_green.png" alt="" />
                   </span>
+                  <a href="config_whitespace.html#WhitespaceAfter">
+                    WhitespaceAfter</a>
+                  <br />
+                  <br />
+                  <span class="wrapper inline">
+                    <img src="images/ok_green.png" alt="" />
```

Comment:
@romani ping

I don't feel this should marked as fully supported. According to https://github.com/checkstyle/checkstyle/pull/7988#discussion_r404379953 other tokens could be added which Google specifies.

Reply:
Yes, I guess you're right. Should I change it? @rnveach

Reply:
if we can not 100% reliably validate it , it is better to make it "blue", in issue please make a comment on what cases we can miss or mishandle.

Reply:
yes @shashwat70

Reply:
Done. @rnveach